test/unit/channel.js

```diff
@@ -478,6 +481,82 @@ describe('Channel _handleChannelEvent', function () {
 		expect(channel.data.hidden).eq(true);
 	});
 
+	it('should update the frozen flag and reload channel state to update `own_capabilities`', () => {
+		const event = {
+			channel: { frozen: true },
+			type: 'channel.updated',
+		};
+		channel.data.frozen = false;
+		sinon.spy(channel, 'query');
```

Comment:
@szuperaz shouldn't we restore the spy? I could not find any restore call here.

Reply:
I've checked and the spy only lives in the scope of the given `it` function, so it doesn't affect other tests.